src/mbgl/map/zoom_history.hpp

```diff
@@ -13,19 +13,20 @@ struct ZoomHistory {
     bool first = true;
 
     bool update(float z, const TimePoint& now) {
+        static TimePoint zero = TimePoint(Duration::zero());
```

Comment:
No need to make this static; likely the compiler can replace `TimePoint(Duration::zero());` with an inline 0 value. You can use `constexpr` as an alias instead.